### PR TITLE
Library

### PR DESCRIPTION
**Pull request by nganifaith**
This is a basic library project which allows a user to add books, delete a book and mark a book as read.

### CODE REVIEW

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hello @nganifaith,
**You have done a great job on this project :muscle:**
### Highlights
- A book can be created :100: 
- A book can be deleted :100: 
- Well done on the styling :clap: 
- Great job on the readme and pr :100: 

### To improve
- [x] When adding a new book, the form should allow us to mark the book `status` as `Read` ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/new_book_button_requirement.png))
- [x] When the book status is `Read`, I should be able to change it to `Unread` by clicking and vise versa ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/toggle_requirement.png))


**Kindly make the above changes before requesting another review**

Feel free to reach me on slack or leave a comment while mentioning me and I will  be happy to respond

_Slack: Oloo Moses_

**Comment by nganifaith** [comment]
Thank you for your timely review. I have done the required changes.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @nganifaith 👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- [x] As stated in the previous review the user should be able to toggle the read status multiple times not just once as is the case now. Kindly fix this as presently it's not possible to toggle the status back to unread.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  books.forEach((book, ind) => renderBook(book, ind));
+}
+
+function addBook(title, author, pages, summary, read = false) {
+  books.push(new Book(title, author, pages, summary, read));
+  getBooks();
+}
+
+function toggle(ind) {
+  books[ind].read = true;
```
Comment (mikenath223) [inline index.js]: - [ ] OPTIONAL - Although, this is a requirement but has been marked as optional as it wasn't mentioned in the previous review. Apart from the Book constructor function consider changing all other functions to es6 arrow function syntax.

**Comment by mikenath223** [comment]
- [ ] OPTIONAL - Also you can consider adding form validation to prevent the user from adding an empty book.

**Comment by mikenath223** [comment]
- [ ] OPTIONAL - In addition, you could also add persistence with localStorage to store the books in the localStorage.

**Comment by nganifaith** [comment]
Hello, @mikenath223 thanks for the review. The user can toggle as many times as possible. Please can you show me what you see? Thank you

**Comment by nganifaith** [comment]
The @mikenath223, however, the toggle option was implemented before I submitted it for review. Please can you kindly check it again and consider clearing your cache just in case. Thank you.

File: index.js (hunk 1 above)
Comment (nganifaith) [reply index.js]: With respect to this, I will work on it when I get more versed with JS. Using ES6 was optional. image

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow. 💯 
  - App is functionsl and available online. 💯 

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**